### ISSUE TITLE
Problems running on docker for windows

### ISSUE THREAD

**Issue by Froghut** [tubearchivist/jellyfin#11]
When I try to run this I get the following error:
```
channel not found in TA: http://tubearchivist:8000/api/channel/X:\Youtube\<channelID>/
[show][UCp3NGUmMHlYQdFuCNEM9W7A] indexing 20 videos
Traceback (most recent call last):
  File "/app/main.py", line 18, in <module>
    main()
  File "/app/main.py", line 14, in main
    library.validate_series()
  File "/app/src/series.py", line 39, in validate_series
    folders: list[str] | None = show_handler.validate_episodes()
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/src/series.py", line 160, in validate_episodes
    ta_video: TAVideo = episode_handler.get_ta_video()
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/src/episode.py", line 18, in get_ta_video
    ta_video: TAVideo = TubeArchivist().get_video(self.youtube_id)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/src/connect.py", line 78, in get_video
    raise ValueError(f"video not found in TA: {url}")
ValueError: video not found in TA: http://tubearchivist:8000/api/video/X:\Youtube\<channelID>\-<videoID>/
```
As the title implies I'm running this in docker on Windows - I mount the youtube media folder in tubearchivist and tubearchivist-jflike this:
```
    volumes:
      - x:/Youtube:/youtube
```

**Comment by bbilly1** [comment]
Windows is untested. Please contribute to make this work on additional platforms.

**Comment by Froghut** [comment]
I created a pull request #12 , I tested my changes on my docker for windows setup and it ran successfully.

**Comment by bbilly1** [comment]
thanks for looking into this. image is rebuild and your fix is in v0.1.1.
